RAW/RawModel.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  RawModel.swift
+//  RAW
+//
+//  Created by Bharat malhotra on 13/07/18.
+//  Copyright © 2018 Bharat malhotra. All rights reserved.
+//
+
+import Foundation
+
+struct RawModel: Codable {
+    
```

Comment:
why is it called rawmodel?

Reply:
Named it randomly, what should be the name?